### PR TITLE
Feature/property pane renovate scroll fix

### PR DESCRIPTION
**Pull request by ram-primathon**
## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/pages/Editor/PropertyPane/index.tsx [hunk 1]
```diff
@@ -46,6 +46,10 @@ const PropertyPaneWrapper = styled(PaneWrapper)<{ themeMode?: EditorTheme }>`
   text-transform: none;
   ${(props) =>
     props.themeMode === EditorTheme.DARK ? scrollbarDark : scrollbarLight};
+
+  &::-webkit-scrollbar {
+    background-color: transparent;
+  }
```
Comment (hetunandu) [inline app/client/src/pages/Editor/PropertyPane/index.tsx:52]: Can we apply this in the scrollbar styles directly?

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/src/pages/Editor/PropertyPane/index.tsx (hunk 1 above)
Comment (ram-primathon) [reply app/client/src/pages/Editor/PropertyPane/index.tsx]: done

File: app/client/src/pages/Editor/PropertyPane/index.tsx [hunk 2]
```diff
@@ -183,14 +187,20 @@ class PropertyPane extends Component<PropertyPaneProps, PropertyPaneState> {
...
+        />
+      );
     return (
       <PropertyPaneWrapper
         themeMode={this.getTheme()}
         ref={this.panelWrapperRef}
         onClick={(e: any) => {
           e.stopPropagation();
         }}
+        backgroundColor={Colors.TRANSPARENT}
```
Comment (hetunandu) [inline app/client/src/pages/Editor/PropertyPane/index.tsx:203]: this is a different fix. Why did we do it like this instead? That fix worked just needed to be part of the scrollbarLight

**Review by hetunandu — CHANGES_REQUESTED** [review]

**Comment by hetunandu** [comment]
This is rendering the scrollbar outside the property pane

**Review by hetunandu — APPROVED** [review]

**Pull request merged**